frontend/my-app/src/app/home/home.component.html

```diff
@@ -0,0 +1,34 @@
+<div id="container">
+  <div id="menu">
+    <app-menu></app-menu>
+  </div>
+  <div id="photos">
+    <ngb-carousel *ngIf="images">
+      <ng-template ngbSlide>

```

Comment:
<ng-template ngbSlide *ngFor="let image of images; let counter = index">
        <div class="picsum-img-wrapper">
          <img [src]="image" alt="Random {{counter+1}} slide">
        </div>
        <div class="carousel-caption">
        </div>
      </ng-template>